### PR TITLE
[enhancement] Update Elasticsearch + verify_certs connection option

### PR DESCRIPTION
**Pull request by syepes**
# Description

- Adds support for the connection option `verify_certs`
  https://www.elastic.co/guide/en/elasticsearch/client/python-api/current/config.html#_verifying_server_certificates

- Updates the Elasticsearch Python lib to 8.15.x as this adds support for the ES|QL Language (`client.esql.query`). This by itself is a game changer when loading data from ES as it can be converted directly into a Data frame 🚀

Example:
```py
@data_loader
def load_data(*args, **kwargs):
    """
    Example of loading data from Elasticsearch using ES|QL
    Returns:
        Data frame
    """

    cfg = {
        'host': get_secret_value('es_src_url'),
        'api_key': get_secret_value('es_src_api_key'),
    }
    filters = {'range': {'@timestamp': {'gte': 'now-1h'}}}
    query = """
    from logs-XYZ
    | LIMIT 3
    """
    df = es_query_esql(cfg, query, filters, **kwargs)

    return df


def es_query_esql(es, query, filter=None, **kwargs) -> DataFrame:
  """
  Query Elasticsearch using SQL and convert the result into a DataFrame
  """
  logger = kwargs.get('logger')

  try:
    # https://www.elastic.co/guide/en/elasticsearch/reference/current/esql-query-api.html
    client = Elasticsearch(es["host"], api_key=es["api_key"])
    resp = client.esql.query(query=query, filter=filter, format="csv")
    df = pd.read_csv(StringIO(resp.body), sep=',')
    logger.info(f"Records: {len(df)}")
    return pd.read_csv(StringIO(resp.body), sep=',')
  except Exception as e:
    logger.error(f"Query: {e}")
    return []

```

# How Has This Been Tested?

- [X] I have tested it locally

# Checklist
- [X] The PR is tagged with proper labels (bug, enhancement, feature, documentation)
- [X] I have performed a self-review of my own code
- [X] I have added unit tests that prove my fix is effective or that my feature works
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation

# cc
@tommydangerous

### CODE REVIEW

File: mage_integrations/mage_integrations/destinations/elasticsearch/target_elasticsearch/sinks.py [hunk 1]
```diff
@@ -185,6 +186,9 @@ def authenticated_client(self) -> elasticsearch.Elasticsearch:
             scheme = "https"
             config["ca_certs"] = self.config[SSL_CA_FILE]
 
+        if scheme == "https":
+            config['verify_certs'] = self.config[VERIFY_CERTS]
```
Comment (wangxiaoyou1993) [inline mage_integrations/mage_integrations/destinations/elasticsearch/target_elasticsearch/sinks.py:190]: I would do `self.config.get(VERIFY_CERTS, True)` here

Comment (syepes) [reply mage_integrations/mage_integrations/destinations/elasticsearch/target_elasticsearch/sinks.py]: Effectively this looks nicer

File: mage_integrations/mage_integrations/destinations/elasticsearch/target_elasticsearch/sinks.py [hunk 2]
```diff
@@ -186,8 +186,7 @@ def authenticated_client(self) -> elasticsearch.Elasticsearch:
             scheme = "https"
             config["ca_certs"] = self.config[SSL_CA_FILE]
 
-        if scheme == "https":
-            config['verify_certs'] = self.config[VERIFY_CERTS]
+        self.config.get(VERIFY_CERTS, True)
```
Comment (wangxiaoyou1993) [inline mage_integrations/mage_integrations/destinations/elasticsearch/target_elasticsearch/sinks.py:189]: I guess you still want to set `config['verify_certs']`
like
```
if scheme == "https":
    config['verify_certs'] = self.config.get(VERIFY_CERTS, True)
```

Comment (syepes) [reply mage_integrations/mage_integrations/destinations/elasticsearch/target_elasticsearch/sinks.py]: Sorry fixed this one

**Review by wangxiaoyou1993 — APPROVED** [review]

**Pull request merged**